tmc/pkg/server/controllers.go

```diff
@@ -403,3 +405,57 @@ func (s *Server) installSyncTargetController(ctx context.Context, config *rest.C
 		return nil
 	})
 }
+
+func (s *Server) installWorkloadReplicateClusterRoleControllers(ctx context.Context, config *rest.Config) error {
```

Comment:
nit: I noticed that some controllers are named `installWorkload...` and some `installWorkloads...`. Would it make sense to unify the names?

Reply:
Created fommowup issue: https://github.com/kcp-dev/kcp/issues/2936